<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinDesktopConfigForm</class>
 <widget class="QWidget" name="KWinDesktopConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>310</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="desktop">
      <attribute name="title">
       <string>Desktops</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Layout</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="numberLabel">
            <property name="whatsThis">
             <string>Here you can set how many virtual desktops you want on your KDE desktop.</string>
            </property>
            <property name="text">
             <string>Number of desktops:</string>
            </property>
            <property name="buddy">
             <cstring>numberSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KIntSpinBox" name="numberSpinBox">
            <property name="whatsThis">
             <string>Here you can set how many virtual desktops you want on your KDE desktop.</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Number of rows:</string>
            </property>
            <property name="buddy">
             <cstring>rowsSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KIntSpinBox" name="rowsSpinBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="activityCheckBox">
            <property name="text">
             <string>Different widgets for each desktop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="nameGroup">
         <property name="title">
          <string>Desktop Names</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="KWin::DesktopNamesWidget" name="desktopNames" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="messageLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="switching">
      <attribute name="title">
       <string>Switching</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="wrapAroundBox">
         <property name="whatsThis">
          <string>Enable this option if you want keyboard or active desktop border navigation beyond the edge of a desktop to take you to the opposite edge of the new desktop.</string>
         </property>
         <property name="text">
          <string>Desktop navigation wraps around</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Desktop Effect Animation</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Animation:</string>
            </property>
            <property name="buddy">
             <cstring>effectComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="KComboBox" name="effectComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KPushButton" name="effectConfigButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KPushButton" name="effectInfoButton"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="popupInfoCheckBox">
         <property name="title">
          <string>Desktop Switch On-Screen Display</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="hideLabel">
            <property name="text">
             <string>Duration:</string>
            </property>
            <property name="buddy">
             <cstring>popupHideSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KIntSpinBox" name="popupHideSpinBox">
            <property name="suffix">
             <string> msec</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="desktopLayoutIndicatorCheckBox">
            <property name="toolTip">
             <string>Enabling this option will show a small preview of the desktop layout indicating the selected desktop.</string>
            </property>
            <property name="text">
             <string>Show desktop layout indicators</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="shortcutsGroupBox">
         <property name="title">
          <string>Shortcuts</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="editorFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="allShortcutsCheckBox">
            <property name="text">
             <string>Show shortcuts for all possible desktops</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KWin::DesktopNamesWidget</class>
   <extends>QWidget</extends>
   <header>desktopnameswidget.h</header>
   <container>1</container>
   <slots>
    <slot>numberChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>numberSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>desktopNames</receiver>
   <slot>numberChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
